tests/unit/client/data-access/UlsLanguageTranslationRepository.spec.ts

```diff
@@ -0,0 +1,28 @@
+import UlsLanguageTranslationRepository from '@/client/data-access/UlsLanguageTranslationRepository';
+import { StringTMap } from '@/datamodel/LanguageTranslations';
+import { WikibaseContentLanguages } from '@/client/MwWindow';
+
+class MockWbWikibaseContentLanguages {
```

Comment:
The class isn't necessary. If we inject a plain object literal, the whole test becomes super simple:
```TypeScript
import UlsLanguageTranslationRepository from '@/client/data-access/UlsLanguageTranslationRepository';

describe( 'UlsLanguageTranslationRepository', () => {
	it( 'gets the language names from the given language translation service', () => {
		const langTranslation = { en: 'English' };
		const translator = {
			getAllPairs() {
				return langTranslation;
			},
		};

		return (
			new UlsLanguageTranslationRepository( translator )
		).getLanguagesInLanguage( 'en' ).then( ( name ) => {
			expect( name ).toEqual( {
				en: langTranslation,
			} );
		} );
	} );
} );
```

Reply:
done